src/typescript/Scala.tmLanguage.ts

```diff
@@ -571,6 +574,26 @@ export const scalaTmLanguage: TmLanguage = {
         }
       }
     },
+    inline: {
+      patterns: [
+        {
+          match: `\\b(inline)\\b(?=(\\s+${plainid}\\s*:)|(.*(val|def)))`,
```

Comment:
It is for the inline arguments (`inline n: Int` in the example). It would also be a `backQuotedId`, I will add it. `plainId` already matches `idUpper`.